text/master-based-development.md

```diff
@@ -0,0 +1,29 @@
+# Master Based Development (MBD)
+- Start Date: Sometime in late 2015 / early 2016
+- RFC PR: https://github.com/CanopyTax/frontend-rfc/pull/1
+
+## Squad vs Chapter vs Frontend-wide
+All squads should use master based development as their branching scheme for deployed services.
+
+## Definition
+Only the master branch may be deployed to integration, staging, and production.
+Other branches only exist for code reviews and should only have one person committing to them. The master branch must
+be production ready at all times, even when a feature is still being worked on.
```

Comment:
Maybe under the definition recognize the alternative term "trunk based development". Also, here is a really good overview that might be worth a link: https://trunkbaseddevelopment.com/

Reply:
👍 will change